app/controllers/mixins/ems_common_angular.rb

```diff
@@ -429,12 +431,14 @@ def set_ems_record_vars(ems, mode = nil)
         default_endpoint = {:role => :default, :hostname => hostname, :port => port}
         default_endpoint.merge!(endpoint_security_options(ems.security_protocol, default_tls_ca_certs))
 
-        if hawkular_hostname.blank?
-          default_key = params[:default_password] || ems.authentication_key
-          hawkular_hostname = get_hostname_from_routes(ems, default_endpoint, default_key)
+        if params[:event_stream_selection] == 'hawkular_enabled'
```

Comment:
If disabled `hawkular_endpoint` stays `{}`.  But it's still [passed into build_connection](https://github.com/moolitayer/manageiq-ui-classic/blob/d84a0ddbf1b71e32fdfd10c93baafbdd84bc16f8/app/controllers/mixins/ems_common_angular.rb#L467).  Did you verify / can you add a test that it will only create 1 Endpoint?